src/app/modules/registration/components/company-details/company-details.component.ts

```diff
@@ -44,6 +50,21 @@ export class CompanyDetailsComponent implements AfterViewInit, OnDestroy {
         patchFormModel(this.formInputs, companyRegistration);
         this.formService.detectChanges(this.formComponent);
       });
+    this.formInputs.sector.options$ = this.store.select(selectSectorOptions)
+      .pipe(filter(nonNull), concatMap(options => {
+        return of([
+          new DynamicFormOption({
+            label: undefined,
+            value: undefined,
+            disabled: true,
+          }),
+          ...options,
+        ]);
+      }));
+    this.formInputs.sector.options$.pipe(takeUntil(this.unsubscriber)).subscribe(() => {
+      this.formInputs.sector.value = undefined;
+    });
```

Comment:
Add comment about Safari bug please, here and for countries

Reply:
done